src/services/redisSessionStorage.ts

```diff
@@ -307,6 +308,14 @@ export default class RedisSessionStorage extends RedisStorageUtils
       .run();
   }
 
+  public async userHasLoginBlocked(
+    fiscalCode: FiscalCode
+  ): Promise<Either<Error, boolean>> {
```

Comment:
`exists` method returns an integer.
From the SDK tests [here](https://github.com/NodeRedis/node-redis/blob/79927c56a0e2fb7819afbb0e13c6078d0015564e/test/commands/exists.spec.js) and from the spec [here](https://redis.io/commands/exists)

Reply:
so there are errors on typos?